src/data/builder-tools.js

```diff
@@ -769,6 +769,14 @@ export const Showcases = [
     website: "https://cexplorer.io",
     getstarted: null,
     tags: ["operatortool", "api"],
+  },
+  {
+    title: "Cardano Leader Slot",
+    description: "Lightweight and Portable Scheduled Blocks Checker for Next, Current and Previous Epochs.",
+    preview: require("./builder-tools/leader-slot.png"),
```

Comment:
The file `./builder-tools/leader-slot.png` must be included with this PR in order for the portal to build.  Please have a look at the other image files in this directory to get an idea what to expect... only a 640p wide sizing of this image is displayed (in a 2:1 aspect ratio) so generally a logo would be fine, rather than a detailed screenshot.

Reply:
I added the image to this PR.